section3/exercises/arrays.rb

```diff
@@ -23,43 +23,51 @@
 
 # YOU DO: Write code below that will print the number of elements in array of
 # animals from above.
+animals.each do |i|
+  puts i
+end
 
 
 # YOU DO: Write code that will reassign the last item in the animals
 # array to "Gorilla"
+p animals[2] = "Gorilla"
+p animals
 
 
-# YOU DO: Write code that will add a new animal (type of your choice) to position 3.
 
+# YOU DO: Write code that will add a new animal (type of your choice) to position 3.
+p animals.insert(2, "Tiger")
 
 # YOU DO: Write code that will print the String "Elephant" in the animals array
-
+p animals << "Elephant"
```

Comment:
✅

Reply:
Though, be cautious if you printed this more than once, how many elephants would be in your array?